test/html.ts

```diff
@@ -16,6 +16,22 @@ describe('render', () => {
     surface = document.createElement('section')
   })
 
+  it('calls `createCallback` on first render', () => {
+    const main = (x = null) => html`<div class="${x}"></div>`
+    let called = false
+    const instance = main()
+    instance.processor = {
+      processCallback() {
+        throw new Error('Expected processCallback to not be called')
+      },
+      createCallback() {
+        called = true
+      }
+    }
+    instance.renderInto(surface)
+    expect(called).to.equal(true)
+  })
```

Comment:
I added this test to be more explicit about the intended behaviour that we don't want `processCallback` called during first pass.